back/app/serializers/web_api/v1/notifications/status_change_on_idea_you_follow_serializer.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-class WebApi::V1::Notifications::StatusChangeOnCommentedIdeaSerializer < WebApi::V1::Notifications::NotificationSerializer
+class WebApi::V1::Notifications::StatusChangeOnIdeaYouFollowSerializer < WebApi::V1::Notifications::NotificationSerializer
```

Comment:
Not really important, but I think the original `...Of...` makes more sense.
```suggestion
class WebApi::V1::Notifications::StatusChangeOfdeaYouFollowSerializer < WebApi::V1::Notifications::NotificationSerializer
```
...though I understand you'd have to change this in a number of places.

Same comment for all other cases where you've changed original `...Of...` to `...On...`

Reply:
Indeed I almost called it `StatusChangeOfIdeaYouFollow` as you suggest, but in the end I felt like I was doubting too much and went for "On" to be consistent with other places (official feedback). I'm not sure if we will have time to change it everywhere, but if I can find the time, I'll do that.

In general, I think it would be nice to have a place to discuss how we name new things in the codebase. Ideally, these discussions should happen before we start coding. As a native English speaker, your input is very valuable on these decisions.

Reply:
I understand. If you don't have much time, then it's not important. It is still clear what it refers to, and it's more important that it works.